scripts/x-data-grid-premium.exports.json

```
@@ -4,20 +4,26 @@
   { "name": "AutoSizerSize", "kind": "Interface" },
   { "name": "bgBG", "kind": "Variable" },
   { "name": "checkGridRowIdIsValid", "kind": "Function" },
+  { "name": "COLUMN_MENU_DEFAULT_COMPONENTS", "kind": "Variable" },
+  { "name": "COLUMN_MENU_DEFAULT_COMPONENTS_PROPS", "kind": "Variable" },
+  { "name": "COLUMN_MENU_SIMPLE_COMPONENTS", "kind": "Variable" },
+  { "name": "COLUMN_MENU_SIMPLE_COMPONENTS_PROPS", "kind": "Variable" },
```

Prefix with `GRID_`?
```suggestion
  { "name": "GRID_COLUMN_MENU_DEFAULT_COMPONENTS", "kind": "Variable" },
```